web/index.php

```diff
@@ -5,6 +5,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Pagerfanta\Pagerfanta;
+use Pagerfanta\Adapter\FixedAdapter;
 use Pagerfanta\Adapter\DoctrineDbalSingleTableAdapter;
```

Comment:
Yeah, in case there is no search query. See https://github.com/outlandishideas/wpackagist/pull/100/files#diff-edf673da361757883b6f7562c08cf9e0R143